pkg/types/azure/tags.go

```diff
@@ -0,0 +1,34 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+)
+
+// GetSystemTags gets the cluster owned system tags.
+func GetSystemTags(clusterID string) map[string]*string {
+	return map[string]*string{
+		fmt.Sprintf("kubernetes.io_cluster.%s", clusterID): to.Ptr("owned"),
```

Comment:
Are there resources where this tag needs to be added with value "shared"?

Reply:
Not that I know of. The only references to shared tags for Azure in the source are for Azure Stack Hub. All of that code is still in place.

Reply:
AFAIK, shared tags are required to add on user-provided vnets when installing cluster in existing vnet, see https://github.com/openshift/installer/pull/7611 for details. 
And FYI, it works well on 4.17+ (capi-based installation), shared tags `"kubernetes.io_cluster.<clusterID>": "shared"` is added on user provided vnet.

Reply:
In testing this, the shared tag still gets added. I haven't modified anything that deals with shared tags.